tests/integration/conftest.py

```diff
@@ -124,3 +196,47 @@ def sample_video() -> str:
     path_to_video = 'tests/integration/media/cat.mp4'
     assert os.path.exists(path_to_video)
     return path_to_video
+
+
+@pytest.fixture
+def organization(client):
+    # Must have at least one seat open in your org to run these tests
```

Comment:
is this gonna make int. tests flakier?

Reply:
No it clears invites before and after (only @labelbox.com invites so users don't accidentally mess up anything). Our test account is only used for testing so there shouldn't be any problem here. This will only matter if you are testing locally and don't have any open seats.